js/zaif.js

```diff
@@ -158,27 +158,43 @@ module.exports = class zaif extends Exchange {
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
             const symbol = base + '/' + quote;
-            const precision = {
-                'amount': -Math.log10 (this.safeNumber (market, 'item_unit_step')),
-                'price': this.safeInteger (market, 'aux_unit_point'),
-            };
             const fees = this.safeValue (this.options['fees'], symbol, this.fees['trading']);
-            const taker = fees['taker'];
-            const maker = fees['maker'];
+            const itemUnitStep = this.safeString (market, 'item_unit_step');
             result.push ({
                 'id': id,
                 'symbol': symbol,
                 'base': base,
                 'quote': quote,
+                'settle': undefined,
                 'baseId': baseId,
                 'quoteId': quoteId,
+                'settleId': undefined,
                 'type': 'spot',
                 'spot': true,
+                'margin': undefined,
+                'swap': false,
+                'future': false,
+                'option': false,
                 'active': true, // can trade or not
```

Comment:
i think we should set active to undefined if it is not in the market response

Reply:
I agree with that. The same is true for currencies.